src/main/java/org/jbake/template/DelegatingTemplateEngine.java

```diff
@@ -30,8 +32,21 @@
     public DelegatingTemplateEngine(final CompositeConfiguration config, final ContentStore db, final File destination, final File templatesPath) {
         super(config, db, destination, templatesPath);
         this.renderers = new TemplateEngines(config, db, destination, templatesPath);
+        this.registerExtractorsForCustomTypes();
```

Comment:
Was thinking about this.... could this call happen when other types are registered by Oven in DocumentTypes or was there a specific reason this occurs here?